tests/integration/runner

```diff
@@ -317,11 +317,18 @@ if __name__ == "__main__":
         "--count", action="store", type=int, dest="count", help="Repeat count"
     )
 
+    parser.add_argument(
+        "--random-order",
+        action="store_true",
+        dest="random_order",
+        help="Run tests in random order using pytest-random-order",
+    )
+
     parser.add_argument(
         "--no-random",
         action="store",
         dest="no_random",
-        help="Disable tests order randomization",
+        help="No-op, use --random-order",
```

Comment:
Do we still use this option somewhere in scripts?

Reply:
Well, it's worth completely getting rid of it, although I'd prefer to have it deprecated for some time.